packages/rlp/test/official.spec.ts

```diff
@@ -148,9 +148,6 @@ const gethCases = [
 
   { input: 'C3808005', value: ['', '', '05'] },
   { input: 'C50183040404', value: ['01', '040404'] },
-  {
-    input: 'c330f9c030f93030ce3030303030303030bd303030303030',
```

Comment:
This one is gone?

Reply:
Its value was missing, I went looking for it but it seems to have always have been missing.

I got an error trying to decode it:

```
❯ node bin/rlp decode 0xc330f9c030f93030ce3030303030303030bd303030303030
Error decoding RLP: invalid RLP (safeSlice): end slice of Uint8Array out-of-bounds
```